### PR TITLE
doc: improve README

### LINKED ISSUE #2: Please add a link to the main project
https://github.com/loeffel-io/ls-lint

This repository is about the action, ls-lint is mentioned, but there is no link

### PR DESCRIPTION
**Pull request by ccoVeille**
Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 # ls-lint - github action
 
-An extremely fast directory and filename linter - Bring some structure to your project filesystem
+This repository is the GitHub action that brings [ls-lint](https://github.com/loeffel-io/ls-lint) to your project easily.
```
Comment (loeffel-io) [inline README.md]: please swap line 5 with 7

File: README.md [hunk 2]
```diff
@@ -26,7 +31,7 @@ default: `false`
 
 ### `debug`
 
-write debug informations to stdout
+write debug information to stdout
```
Comment (loeffel-io) [inline README.md:35]: i would like to use `informations` because debug prints skips, statistics etc

**Review by loeffel-io — CHANGES_REQUESTED** [review]
LGTM and good idea - just some mini changes please

File: README.md (hunk 2 above)
Comment (ccoVeille) [reply README.md:35]: information is uncountable, so it has no plural. It's often misused. That's why I fixed it.

https://dictionary.cambridge.org/grammar/british-grammar/information

But I can rollback, if you prefer.

Please tell me.

File: README.md (hunk 1 above)
Comment (ccoVeille) [reply README.md]: Sure

Comment (ccoVeille) [reply README.md]: I'll move the link to ls-lint on content currently line 7, then

**Review by loeffel-io — APPROVED** [review]

**Pull request merged**